p2pool/src/server/http/stats_collector.rs

```diff
@@ -15,6 +16,15 @@ use tokio::{
     time::MissedTickBehavior,
 };
 
+#[derive(Clone)]
+pub struct PeerStats {
+    pub peer_is_a_seed_peer: bool,
+    pub peer_id: PeerId,
+    pub public_addresses: Vec<Multiaddr>,
+    pub number_received: u64,
```

Comment:
```suggestion
    pub number_received: Option<u64>,
```

Reply:
The option will not work here as the number_received will never be None, it will always be 0 or >0, where 0 is a valid number.